tests/Operation/ListCollectionsFunctionalTest.php

```diff
@@ -34,6 +34,30 @@ public function testListCollectionsForNewlyCreatedDatabase()
         }
     }
 
+    public function testOffsetGet()
+    {
+        if (version_compare($this->getServerVersion(), '3.4.0', '<')) {
+            $this->markTestSkipped('idIndex and info are not supported');
+        }
```

Comment:
I don't see any harm in keeping this for testing 3.4+ servers, but I'd suggest renaming it from `testOffsetGet()` to something like `testIdIndexAndInfo()` instead.